store/iavl/store.go

```diff
@@ -23,7 +24,7 @@ import (
 )
 
 const (
-	defaultIAVLCacheSize = 10000
+	DefaultIAVLCacheSize = 500000
```

Comment:
Is the number we used in later versions? (Just want to sanity check)

Reply:
I'll check that out, too.  I think that the answer is yes.

Reply:
yess, it is